crmsh/utils.py

```diff
@@ -999,7 +999,34 @@ def get_dc(peer=None):
     return out.split()[-1]
 
 
-def wait4dc(what="", show_progress=True):
+def wait_for_dc(node: str = None):
+    """
+    Wait for the cluster's DC to become available
+    """
+    if not ServiceManager().service_is_active("pacemaker.service", remote_addr=node):
+        raise ValueError("Pacemaker is not running. No DC.")
+    dc_deadtime = get_property("dc-deadtime", peer=node) or str(constants.DC_DEADTIME_DEFAULT)
+    dc_timeout = int(dc_deadtime.strip('s')) + 5
```

Comment:
`dc-deadtime` value  takes `time` and it could be `s|m|h` and maybe even `d` days.